open-company/open-company-web
# Auto draft

Requirements doc: https://docs.google.com/document/d/1TTJdi_ajlrUkotDhVL1qT1smuSFH22zzcAoqdBkPvjo/edit
Trello: 
https://trello.com/c/Sc2g1qL4/1447-saving-saved-server-side-not-just-with-cookie

When a user creates a draft, these changes will automatically save any changes made. Both on the server side and in the client.

Depends on https://github.com/open-company/open-company-storage/pull/142

Test steps:

Test edit view
- click 'New'
- type in some changes
- [ ] Do you see the 'Saving..' 'Saved' message?
- close the browser window
- [ ] after reloading you should see a draft with your changes
***


- edit the draft
- add some changes
- [ ] Do you see the 'Saving..' 'Saved' message?
- click the x
- click 'lose changes'
- [ ] is the original post restored?

***
- click 'New'
- add some changes
- [ ] Do you see the 'Saving..' 'Saved' message?
- click x
- [ ] is the post deleted?
***

- click 'New'
- make some changes
- click save
- [ ] is there a draft?

***
- click 'New'
- make some changes
- click Post
- [ ] is there a post?

***
- click 'New'
- save a draft
- edit the draft
- click Post
- [ ] is there a post?

***
Test full screen edit
Normal full screen edit should function as before. Just go through some edits to ensure it still works.
